sumatra/commands.py

```diff
@@ -224,8 +224,8 @@ def configure(argv):
                             description=description)
     parser.add_argument('-d', '--datapath', metavar='PATH', help="set the path to the directory in which smt will search for datafiles generated by the simulation or analysis.")
     parser.add_argument('-i', '--input', metavar='PATH', default=None, help="set the path to the directory in which smt will search for input datafiles.")
-    parser.add_argument('-l', '--addlabel', choices=['cmdline', 'parameters', None], metavar='OPTION',
-                        default=None, help="If this option is set, smt will append the record label either to the command line (option 'cmdline') or to the parameter file (option 'parameters'), and will add the label to the datapath when searching for datafiles. It is up to the user to make use of this label inside their program to ensure files are created in the appropriate location.")
+    parser.add_argument('-l', '--addlabel', choices=['cmdline', 'parameters', 'none'], metavar='OPTION',
```

Comment:
Given that a lot of Sumatra users probably use Python, does it make sense to allow `'None'` in addition to `'none'` because Python users would expect this to work? Cf. the example by OP in bug report #316.